chromadb/test/db/test_system.py

```diff
@@ -98,8 +98,24 @@ def grpc_with_mock_server() -> Generator[SysDB, None, None]:
     yield client
 
 
+def grpc_with_real_server() -> Generator[SysDB, None, None]:
```

Comment:
we can comment out this fixture for now and we will get CI running against this service later this week!